### ISSUE TITLE
bug: Uncaught (in promise) TypeError: request.clone is not a function

### ISSUE THREAD

**Issue by mobeigi** [requestly/requestly#1295]
### Describe your issue?

With the Requestly chrome extension enabled, the follow website fails to load:
https://cubari.moe/read/gist/JYHJU/001/1/

The console contains the following logs:
![image](https://github.com/requestly/requestly/assets/5266113/5520fc60-fd4b-425b-985c-d8d16c1c7098)

The errors go away when the plugin is disabled via Chrome.
Even if Requestly is paused and no rules exist, the issue occurs.

### Repro steps

1. Install and enable the Request app
2. Visit website: https://cubari.moe/read/gist/JYHJU/001/1/
3. Page will fail to load and errors are in the console.

### What Requestly tool were you using?

- [X] Extension
- [ ] Desktop-App
- [ ] Android SDK
- [ ] Selenium

### Your Environment

Windows 10, Chrome Version 120.0.6099.111 (Official Build) (64-bit)

### Requestly Version

23.12.12

### Error screenshot

![image](https://github.com/requestly/requestly/assets/5266113/5520fc60-fd4b-425b-985c-d8d16c1c7098)

**Comment by nafees87n** [comment]
@mobeigi 
Thanks for reporting this issue. We are investigating this and will try to fix this as soon as possible.

**Comment by nafees87n** [comment]
This looks like a bug on https://cubari.moe/. The `Request` object is overridden by a page script which is the cause of this error.
<img width="1091" alt="Screenshot 2024-01-02 at 12 04 53 PM" src="https://github.com/requestly/requestly/assets/56021937/4a00009d-b17e-411d-8398-1816bed6c9f9">

**Comment by mobeigi** [comment]
@nafees87n Are you able to reproduce the error or bug with Requestly disabled? I am only able to produce the error with the Requestly extension on.

So whatever Requestly is trying to do is interfering with what the website is doing.
Also I'm not sure what `alg_lib` is, where its from and what its purpose is.

**Comment by nafees87n** [comment]
Yeah. Requestly uses the `Request` object in page script. We will try to fix it in the next release so that Requestly does not block the website.

**Comment by nafees87n** [comment]
@mobeigi Can you please confirm if you still get this error in the latest Requestly version `24.7.7`?

**Comment by mobeigi** [comment]
> @mobeigi Can you please confirm if you still get this error in the latest Requestly version `24.7.7`?

Yeah, still getting same issue with 24.7.7.

![image](https://github.com/user-attachments/assets/f8f80359-eccb-4bad-8177-19aabb616c65)

**Comment by Ronakkadhi** [comment]
Closing as stale — no activity for over a year.

As part of the 2026 repository restructure, we're closing dormant issues to focus on actively-tracked items. If this is still relevant, please re-open or file a fresh issue at the right home:

- 🌐 **HTTP Interceptor** (Chrome extension + web app + desktop proxy) → [requestly/interceptor](https://github.com/requestly/interceptor/issues/new/choose)
- 🚀 **Requestly API Client** → this repo

Thanks for taking the time to file the original report.
